README.md

```diff
@@ -137,7 +137,10 @@ git clone https://huggingface.co/csukuangfj/icefall-asr-librispeech-pruned-state
   --nn-model-filename ./icefall-asr-librispeech-pruned-stateless-emformer-rnnt2-2022-06-01/exp/cpu_jit-epoch-39-avg-6-use-averaged-model-1.pt \
   --bpe-model-filename ./icefall-asr-librispeech-pruned-stateless-emformer-rnnt2-2022-06-01/data/lang_bpe_500/bpe.model
 ```
-
+Here, before running the web client, you also need to map your server ports to your local ports in the server terminal with the following command:
```

Comment:
Please also add that
(1) You only need to do this if the asr server is running on a machine different from the client
(2) The command is run in the terminal on the server machine